src/test/rustdoc/decl_macro.rs

```diff
@@ -18,8 +18,8 @@ pub macro my_macro_2($($tok:tt)*) {
 
 // @has decl_macro/macro.my_macro_multi.html //pre 'pub macro my_macro_multi {'
 // @has - //pre '(_) => { ... },'
-// @has - //pre '($foo:ident . $bar:expr) => { ... },'
-// @has - //pre '($($foo:literal),+) => { ... }'
+// @has - //pre '($ foo : ident.$ bar : expr) => { ... },'
+// @has - //pre '($ ($ foo : literal), +) => { ... },'
```

Comment:
This is less readable IMO. The `$foo:ident` is semantically a single entity. I have never seen anyone write it with spaces inbetween.

Reply:
See https://github.com/rust-lang/rust/pull/86282#issuecomment-860296914.